pyanaconda/core/util.py

```diff
@@ -334,15 +334,16 @@ def _run_program(argv, root='/', stdin=None, stdout=None, env_prune=None, log_ou
     return (proc.returncode, output_string)
 
 
-def execInSysroot(command, argv, stdin=None):
+def execInSysroot(command, argv, stdin=None, root=getSysroot()):
```

Comment:
The sysroot can change during the installation. You should set it inside the function.

Reply:
Yes, ideally just use `root=None` in the header and if `None` set the sysroot in the function body.

Reply:
Fixed. :)